src/bin/bat/config.rs

```diff
@@ -16,8 +16,22 @@ pub fn config_file() -> PathBuf {
 }
 
 pub fn get_args_from_config_file() -> Result<Vec<OsString>, shell_words::ParseError> {
+    let mut config_file_path = vec![PathBuf::from("/etc/bat/config")];
+    config_file_path.push(default_config_path());
+
+    let mut config_args = vec![];
+
+    for path in config_file_path.iter() {
+        let args = read_args_from_config_file(path.to_path_buf());
+        config_args.append(&mut args.unwrap());
+    }
```

Comment:
It's not necessary, but the whole loop could probably be written in functional style with `.map(…)` and `.collect()`. This would also avoid the `mut` on `config_args`.

Something like (not tested and most probably wrong):
``` rust
let config_args = config_file_path
  .iter()
  .map(read_args_from_config_file)
  .transpose()?
  .flatten()
 .collect();
```